nucypher/cli/actions.py

```diff
@@ -76,6 +96,13 @@ def destroy_configuration_root(config_root=None, force=False, logs: bool = False
     return config_root
 
 
+def get_external_ip():
+    ip_request = requests.get('https://ifconfig.me/')
+    if ip_request.status == 200:
```

Comment:
Is this supposed to be `status_code`?

Reply:
Yeah - we need a bi of coverage here - see #971